tests/chainer_tests/links_tests/caffe_tests/test_caffe_function.py

```diff
@@ -139,9 +139,9 @@ class TestConvolution(TestCaffeFunctionBaseMock):
                 'bottom': ['x'],
                 'top': ['y'],
                 'convolution_param': {
-                    'kernel_size': 2,
-                    'stride': 3,
-                    'pad': 4,
+                    'kernel_size': [2],
+                    'stride': [3],
+                    'pad': [4],
```

Comment:
I have question that, after updating for new prototxt spec, existing caffe models in modelzoo such as alexnet, caffenet and googlenet  work as before if they use ConvolutionLayer? If they do not work, we unfortunately have to manage caffe_function.py and caffe_pb{2,3}.py for each version of caffe.